src/Core/Models/StaticStore/Plans/TeamsStarterPlan.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Bit.Core.Enums;
+
+namespace Bit.Core.Models.StaticStore.Plans;
+
+public record TeamsStarterPlan : Plan
+{
+    public TeamsStarterPlan()
+    {
+        Type = PlanType.TeamsStarter;
+        Product = ProductType.TeamsStarter;
+        Name = "Teams (Starter)";
+        NameLocalizationKey = "planNameTeamsStarter";
+        DescriptionLocalizationKey = "planDescTeams";
+        CanBeUsedByBusiness = true;
+
+        TrialPeriodDays = 7;
+
+        HasGroups = true;
+        HasDirectory = true;
+        HasEvents = true;
+        HasTotp = true;
+        Has2fa = true;
+        HasApi = true;
+        UsersGetPremium = true;
+
+        UpgradeSortOrder = 2;
```

Comment:
Are customer able to upgrade from TeamsStarterPlan to Enterprise

Reply:
No, not yet at least. That will be included in the December release